Remove personal (#204)
codesandbox/docs: packages/projects-docs/pages/learn/plans/workspace.mdx

![Create a team](../images/workspace-selector-newteam.png)

## Managing workspaces and subscriptions

You can create and manage workspaces from the dashboard settings page.
To find this, select your workspace and open the subscription page.

![Team settings](../images/workspace-team-settings-btn.png)

Here you can see the team overview with subscription details, team members and roles.
Admins can edit team member access and roles here as well as edit the subscription.

![Team settings](../images/workspace-team-settings.png)

## Transitioned personal workspaces

As of Nov 12, 2023, all personal workspaces act as standard workspaces.
Previously, everyone who signed up for a CodeSandbox account would automatically have a personal workspace created for them.
This workspace would hold:

- your account details
- your contribution branches
- all the sandboxes created in the workspace

However, there was no option to add people to the workspace or upgrade the space.
With the recent changes, every workspace will be treated the same. This means that workspaces that were previously a Free Personal can do the following:

#### Add collaborators

> Just like in a Pro workspace, you may add members to the workspace by navigating to the workspace settings, and either sending an invite URL or entering an email address that will send off an invitation.

Reviewer: ```suggestion
Just like in a Pro workspace, you may add members to the workspace by navigating to the workspace settings, and either sending an invite URL or entering the email address of the member you wish to add (they will be notified by email).
```

#### Upgrade to Pro

> Every workspace can now be upgraded. By selecting any action to upgrade from within a workspace, you will be able

Reviewer: Something missing here @necoline

Author: Thanks, I finished the sentence here
